recipe/meta.yaml

```diff
@@ -30,27 +26,35 @@ requirements:
     - libgcrypt
     - libgpg-error
     - libiconv  # [not linux]
-    - libtool
-    - libxml2
-    - libxslt
+    - libtool    
```

Comment:
that is C/C++ shouldn't we have explicit pinning ?

Reply:
can you elaborate on that ? until now  I though that all C/C++ packages needs to have pinning, but now I got confused as it seems to be extra rules

Reply:
The linter is not accurate in this case. It looks like a linter bug

Reply:
so the extra rule is that even if it is C/C++ but does not have run_exports then it does not need pinning, right ? @marcoesters is it something we can improve linter on ?

Reply:
The linter is not a closed project, so PRs are always welcome: https://github.com/anaconda-distribution/anaconda-linter